xml/installation-installation-configure_lbaas.xml

```diff
@@ -232,482 +233,363 @@
+| OS-EXT-SRV-ATTR:instance_name       | instance-0000001f                                          |
+| OS-EXT-STS:power_state              | Running                                                    |
+| OS-EXT-STS:task_state               | None                                                       |
+| OS-EXT-STS:vm_state                 | active                                                     |
+| OS-SRV-USG:launched_at              | 2019-10-23T13:36:32.000000                                 |
+| OS-SRV-USG:terminated_at            | None                                                       |
+| accessIPv4                          |                                                            |
+| accessIPv6                          |                                                            |
+| addresses                           | lb_net2=12.12.12.13                                        |
+| adminPass                           | 79GcsJQfNt3p                                               |
+| config_drive                        |                                                            |
+| created                             | 2019-10-23T13:36:17Z                                       |
+| flavor                              | m1.tiny (1)                                                |
+| hostId                              | ed0511cb20d44f420e707fbc801643e0658eb2efbf980ef986aa45d0   |
+| id                                  | 874e675b-727b-44a2-99a9-0ff178590f86                       |
+| image                               | cirros-0.4.0-x86_64 (96e0df13-573f-4672-97b9-3fe67ff84d6a) |
+| key_name                            | id_rsa2                                                    |
+| name                                | lb2_vm1                                                    |
+| progress                            | 0                                                          |
+| project_id                          | de095070f242416cb3dc4cd00e3c79f7                           |
+| properties                          |                                                            |
+| security_groups                     | name='letmein'                                             |
+| status                              | ACTIVE                                                     |
+| updated                             | 2019-10-23T13:36:32Z                                       |
+| user_id                             | ec4d4dbf03be4bf09f105c06c7562ba7                           |
+| volumes_attached                    |                                                            |
++-------------------------------------+------------------------------------------------------------+</screen>
+    <para>
+     Save the private IP of the first web server
+     <screen>&prompt.ardana;IP_LBVM1="12.12.12.13"</screen>
+    </para>
+    <para>
+     Boot second "webserver"
```

Comment:
```suggestion
     Boot  a second webserver:

Reply:
Why is webserver in inverted commas?

Reply:
Inverted commas ? its supposed to be quotes. But the web server is only a bash script that is responding to traffic on port 80, so its acting  like a web server in a limited fashion.  I'll go for pseudo  web server

Reply:
removed the quotes everywhere, only put pseudo in one or two places though, maybe that needs another update?